src/openzeppelin/tests/mocks/reentrancy_mock.cairo

```diff
@@ -1,93 +1,92 @@
 use starknet::ContractAddress;
 
-#[abi]
-trait IReentrancyGuarded {
-    fn count_external_recursive(n: felt252);
+#[starknet::interface]
+trait IReentrancyGuarded<TContractState> {
+    fn count_external_recursive(ref self: TContractState, n: felt252);
 }
 
-#[abi]
-trait IReentrancyMock {
-    #[view]
-    fn current_count() -> felt252;
-    #[external]
-    fn callback();
-    #[external]
-    fn count_local_recursive(n: felt252);
-    #[external]
-    fn count_external_recursive(n: felt252);
-    #[external]
-    fn count_and_call(attacker: ContractAddress);
-    #[external]
-    fn count();
+#[starknet::interface]
+trait IReentrancyMock<TContractState> {
```

Comment:
```suggestion
trait IReentrancyMock<TState> {
```

I think we should use this in mock as well for consistency.

Reply:
Good call, will update